src/public/translations/en/translation.json

```diff
@@ -1401,5 +1403,26 @@
   },
   "sql_table_schemas": {
     "tables": "Tables"
+  },
+  "tab_row": {
+    "close_tab": "Close tab",
+    "add_new_tab": "Add new tab",
+    "close": "Close",
+    "close_other_tabs": "Close other tabs",
+    "close_all_tabs": "Close all tabs",
+    "move_tab_to_new_window": "Move this tab to a new window",
+    "new_tab": "New tab"
+  },
+  "toc": {
+    "table_of_contents": "Table of Contents",
+    "options": "Options"
+  },
+  "watched_file_update_status": {
+    "file_last_modified": "File <code class=\"file-path\"></code> has been last modified on <span class=\"file-last-modified\"></span>.",
```

Comment:
@Nriver , would it be possible for you to look into interpolation without string escape for this?
I would really like not having HTML inside the translations.

Reference: https://www.i18next.com/translation-function/interpolation#unescape

Reply:
I appreciate the suggestion, but I prefer not to pursue that. It adds significant work with minimal benefit.